fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -418,7 +418,7 @@ interval
     ;
 
 unitIdentifier
-    : YEAR | MONTH | WEEK | DAY | HOUR | MINUTE | SECOND
+    : YEAR | MONTH | WEEK | DAY | HOUR | MINUTE | SECOND | MICROSECOND
```

Comment:
do not add this, if u do not want to add MICROSECOND interval literal

Reply:
It has been removed, but I will implement it next